app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+    if len(command_parts) != 3:
+        raise ValueError(f"Expected 3 arguments, got {len(command_parts)}")
+
+    command, old_file, new_file = command_parts
+    if old_file == new_file:
```

Comment:
Your approach with raising Exceptions looks well structured however in this task  I would suggest for this task to use conditional statements to check that command is correct. You can structure conditionals in a way that if conditions are met you continue with the rest of code, otherwise nothing will be executed and the program will exit. This will save you couple of lines of code and you may write less instructions